version

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
all other services start with 0.1.0, let's also do that here

Reply:
agreed. just made a PR for that